tools/run-dev.py

```diff
@@ -305,7 +305,6 @@ class Application(web.Application):
     def __init__(self):
         # type: () -> None
         handlers = [
-            (r"/sockjs/.*/websocket$", TornadoHandler),
```

Comment:
What's the story with this junk?  Was that just unnecessary given the similar line below?  If so I'd put it in its own commit with the explanation.